'Error compiling public/less/bootstrap-build.less' when npm start.

>npm -version

{ Drywall: '0.16.1',
  npm: '2.11.2',
  http_parser: '2.3',
  modules: '14',
  node: '0.12.4',
  openssl: '1.0.1m',
  uv: '1.5.0',
  v8: '3.28.71.19',
  zlib: '1.2.8' }

=============================
>grunt

Running "copy:vendor" (copy) task
Created 5 directories, copied 109 files

Running "newer:uglify" (newer) task

Running "newer:uglify:layouts" (newer) task
No newer files to process.

Running "newer:uglify:views" (newer) task
No newer files to process.

Running "newer:less" (newer) task

Running "newer:less:layouts" (newer) task

Running "less:layouts" (less) task
>> NameError: variable @btn-border-radius-base is undefined in public/vendor/bootstrap/less/buttons.less on line 20, column 102:
>> 19   white-space: nowrap;
>> 20   .button-size(@padding-base-vertical; @padding-base-horizontal; @font-size-base; @line-height-base; @btn-border-radius-base);
>> 21   .user-select(none);
Warning: Error compiling public/less/bootstrap-build.less Use --force to continue.

================================

And I notice some warn msg when doing 'npm install' related to node-pre-gyp.

```
> bcrypt@0.8.3 install /home/squallssck/drywall/node_modules/bcrypt
> node-gyp rebuild

make: Entering directory `/home/squallssck/drywall/node_modules/bcrypt/build'
  CXX(target) Release/obj.target/bcrypt_lib/src/blowfish.o
  CXX(target) Release/obj.target/bcrypt_lib/src/bcrypt.o
  CXX(target) Release/obj.target/bcrypt_lib/src/bcrypt_node.o
  SOLINK_MODULE(target) Release/obj.target/bcrypt_lib.node
  COPY Release/bcrypt_lib.node
make: Leaving directory `/home/squallssck/drywall/node_modules/bcrypt/build'
npm WARN prefer global grunt-cli@0.1.13 should be installed with -g
 
> kerberos@0.0.12 install /home/squallssck/drywall/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos
> (node-gyp rebuild 2> builderror.log) || (exit 0)

make: Entering directory `/home/squallssck/drywall/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
  CXX(target) Release/obj.target/kerberos/lib/worker.o
  CC(target) Release/obj.target/kerberos/lib/kerberosgss.o
  CC(target) Release/obj.target/kerberos/lib/base64.o
  CXX(target) Release/obj.target/kerberos/lib/kerberos_context.o
  SOLINK_MODULE(target) Release/obj.target/kerberos.node
  COPY Release/kerberos.node
make: Leaving directory `/home/squallssck/drywall/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos/build'
npm WARN installMany nopt was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany npmlog was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany request was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany semver was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany tar was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany tar-pack was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany mkdirp was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany rc was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany rimraf was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany nopt was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany npmlog was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany request was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany semver was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany tar was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany tar-pack was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany mkdirp was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany rc was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
npm WARN installMany rimraf was bundled with node-pre-gyp@0.6.4, but bundled package wasn't found in unpacked tree
 
> kerberos@0.0.12 install /home/squallssck/drywall/node_modules/mongoose/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos
> (node-gyp rebuild 2> builderror.log) || (exit 0)

make: Entering directory `/home/squallssck/drywall/node_modules/mongoose/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
  CXX(target) Release/obj.target/kerberos/lib/worker.o
  CC(target) Release/obj.target/kerberos/lib/kerberosgss.o
  CC(target) Release/obj.target/kerberos/lib/base64.o
  CXX(target) Release/obj.target/kerberos/lib/kerberos_context.o
  SOLINK_MODULE(target) Release/obj.target/kerberos.node
  COPY Release/kerberos.node
make: Leaving directory `/home/squallssck/drywall/node_modules/mongoose/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos/build'
\
> bson-ext@0.1.8 install /home/squallssck/drywall/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/bson/node_modules/bson-ext
> (node-pre-gyp install --fallback-to-build) || (node-gyp rebuild 2> builderror.log) || (exit 0)

make: Entering directory `/home/squallssck/drywall/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/bson/node_modules/bson-ext/build'
  CXX(target) Release/obj.target/bson/ext/bson.o
  SOLINK_MODULE(target) Release/obj.target/bson.node
  COPY Release/bson.node
make: Leaving directory `/home/squallssck/drywall/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/bson/node_modules/bson-ext/build'

> bson-ext@0.1.8 install /home/squallssck/drywall/node_modules/mongoose/node_modules/bson/node_modules/bson-ext
> (node-pre-gyp install --fallback-to-build) || (node-gyp rebuild 2> builderror.log) || (exit 0)

make: Entering directory `/home/squallssck/drywall/node_modules/mongoose/node_modules/bson/node_modules/bson-ext/build'
  CXX(target) Release/obj.target/bson/ext/bson.o
  SOLINK_MODULE(target) Release/obj.target/bson.node
  COPY Release/bson.node
make: Leaving directory `/home/squallssck/drywall/node_modules/mongoose/node_modules/bson/node_modules/bson-ext/build'
jquery.cookie@1.4.1 node_modules/jquery.cookie

async@0.9.2 node_modules/async

underscore@1.8.3 node_modules/underscore

cookie-parser@1.3.5 node_modules/cookie-parser
├── cookie@0.1.3
└── cookie-signature@1.0.6

passport-local@1.0.0 node_modules/passport-local
└── passport-strategy@1.0.0

backbone@1.2.1 node_modules/backbone

grunt-contrib-clean@0.6.0 node_modules/grunt-contrib-clean
└── rimraf@2.2.8

passport@0.2.2 node_modules/passport
├── passport-strategy@1.0.0
└── pause@0.0.1

grunt-concurrent@1.0.1 node_modules/grunt-concurrent
└── pad-stdio@1.0.0 (lpad@1.0.0)

passport-google-oauth@0.2.0 node_modules/passport-google-oauth
└── pkginfo@0.3.0

respond.js@1.4.2 node_modules/respond.js

csurf@1.8.3 node_modules/csurf
├── cookie@0.1.3
├── cookie-signature@1.0.6
├── http-errors@1.3.1 (inherits@2.0.1, statuses@1.2.1)
└── csrf@3.0.0 (rndm@1.1.0, uid-safe@2.0.0, base64-url@1.2.1, scmp@1.0.0)

method-override@2.3.3 node_modules/method-override
├── methods@1.1.1
├── vary@1.0.0
├── parseurl@1.3.0
└── debug@2.2.0 (ms@0.7.1)

morgan@1.6.0 node_modules/morgan
├── basic-auth@1.0.2
├── on-headers@1.0.0
├── depd@1.0.1
├── on-finished@2.3.0 (ee-first@1.1.1)
└── debug@2.2.0 (ms@0.7.1)

express-session@1.11.3 node_modules/express-session
├── cookie@0.1.3
├── utils-merge@1.0.0
├── cookie-signature@1.0.6
├── on-headers@1.0.0
├── parseurl@1.3.0
├── depd@1.0.1
├── uid-safe@2.0.0 (base64-url@1.2.1)
├── crc@3.3.0
└── debug@2.2.0 (ms@0.7.1)

grunt-contrib-copy@0.8.0 node_modules/grunt-contrib-copy
├── file-sync-cmp@0.1.1
└── chalk@0.5.1 (ansi-styles@1.1.0, escape-string-regexp@1.0.3, supports-color@0.2.0, strip-ansi@0.3.0, has-ansi@0.1.0)

grunt-newer@1.1.1 node_modules/grunt-newer
├── rimraf@2.2.8
└── async@0.9.0

serve-static@1.9.3 node_modules/serve-static
├── utils-merge@1.0.0
├── escape-html@1.0.1
├── parseurl@1.3.0
└── send@0.12.3 (destroy@1.0.3, fresh@0.2.4, range-parser@1.0.2, ms@0.7.1, depd@1.0.1, debug@2.2.0, mime@1.3.4, on-finished@2.2.1, etag@1.6.0)

passport-github@0.1.5 node_modules/passport-github
├── pkginfo@0.2.3
└── passport-oauth@0.1.15 (passport@0.1.18, oauth@0.9.13)

passport-tumblr@0.1.2 node_modules/passport-tumblr
├── pkginfo@0.2.3
└── passport-oauth@0.1.15 (passport@0.1.18, oauth@0.9.13)

passport-oauth@1.0.0 node_modules/passport-oauth
├── passport-oauth1@1.0.1 (utils-merge@1.0.0, passport-strategy@1.0.0, oauth@0.9.13)
└── passport-oauth2@1.1.2 (uid2@0.0.3, passport-strategy@1.0.0, oauth@0.9.13)

passport-twitter@1.0.3 node_modules/passport-twitter
├── xtraverse@0.1.0 (xmldom@0.1.19)
└── passport-oauth1@1.0.1 (utils-merge@1.0.0, passport-strategy@1.0.0, oauth@0.9.13)

passport-facebook@2.0.0 node_modules/passport-facebook
└── passport-oauth2@1.1.2 (uid2@0.0.3, passport-strategy@1.0.0, oauth@0.9.13)

compression@1.5.0 node_modules/compression
├── bytes@2.1.0
├── vary@1.0.0
├── on-headers@1.0.0
├── debug@2.2.0 (ms@0.7.1)
├── compressible@2.0.3 (mime-db@1.13.0)
└── accepts@1.2.9 (negotiator@0.5.3, mime-types@2.1.1)

html5shiv@3.7.2 node_modules/html5shiv

express@4.12.4 node_modules/express
├── utils-merge@1.0.0
├── cookie-signature@1.0.6
├── methods@1.1.1
├── fresh@0.2.4
├── escape-html@1.0.1
├── range-parser@1.0.2
├── merge-descriptors@1.0.0
├── content-type@1.0.1
├── vary@1.0.0
├── parseurl@1.3.0
├── cookie@0.1.2
├── finalhandler@0.3.6
├── content-disposition@0.5.0
├── path-to-regexp@0.1.3
├── depd@1.0.1
├── on-finished@2.2.1 (ee-first@1.1.0)
├── qs@2.4.2
├── debug@2.2.0 (ms@0.7.1)
├── etag@1.6.0 (crc@3.2.1)
├── send@0.12.3 (destroy@1.0.3, ms@0.7.1, mime@1.3.4)
├── proxy-addr@1.0.8 (forwarded@0.1.0, ipaddr.js@1.0.1)
├── type-is@1.6.3 (media-typer@0.3.0, mime-types@2.1.1)
└── accepts@1.2.9 (negotiator@0.5.3, mime-types@2.1.1)

body-parser@1.13.1 node_modules/body-parser
├── bytes@2.1.0
├── content-type@1.0.1
├── depd@1.0.1
├── on-finished@2.3.0 (ee-first@1.1.1)
├── http-errors@1.3.1 (inherits@2.0.1, statuses@1.2.1)
├── qs@2.4.2
├── debug@2.2.0 (ms@0.7.1)
├── raw-body@2.1.1 (unpipe@1.0.0)
├── type-is@1.6.3 (media-typer@0.3.0, mime-types@2.1.1)
└── iconv-lite@0.4.10

font-awesome@4.3.0 node_modules/font-awesome

bcrypt@0.8.3 node_modules/bcrypt
├── bindings@1.2.1
└── nan@1.8.4

grunt-contrib-watch@0.6.1 node_modules/grunt-contrib-watch
├── async@0.2.10
├── tiny-lr-fork@0.0.5 (debug@0.7.4, faye-websocket@0.4.4, noptify@0.0.3, qs@0.5.6)
├── gaze@0.5.1 (globule@0.1.0)
└── lodash@2.4.2

grunt-cli@0.1.13 node_modules/grunt-cli
├── resolve@0.3.1
├── nopt@1.0.10 (abbrev@1.0.7)
└── findup-sync@0.1.3 (glob@3.2.11, lodash@2.4.2)

emailjs@0.3.16 node_modules/emailjs
├── bufferjs@1.1.0
├── addressparser@0.3.2
├── starttls@0.2.1
├── mimelib@0.2.14 (addressparser@0.2.1, encoding@0.1.11)
└── moment@1.7.0

jquery@2.1.4 node_modules/jquery

grunt@0.4.5 node_modules/grunt
├── dateformat@1.0.2-1.2.3
├── which@1.0.9
├── getobject@0.1.0
├── eventemitter2@0.4.14
├── rimraf@2.2.8
├── colors@0.6.2
├── hooker@0.2.3
├── async@0.1.22
├── grunt-legacy-util@0.2.0
├── exit@0.1.2
├── nopt@1.0.10 (abbrev@1.0.7)
├── minimatch@0.2.14 (sigmund@1.0.1, lru-cache@2.6.4)
├── glob@3.1.21 (inherits@1.0.0, graceful-fs@1.2.3)
├── lodash@0.9.2
├── coffee-script@1.3.3
├── underscore.string@2.2.1
├── iconv-lite@0.2.11
├── findup-sync@0.1.3 (glob@3.2.11, lodash@2.4.2)
├── grunt-legacy-log@0.1.2 (grunt-legacy-log-utils@0.1.1, underscore.string@2.3.3, lodash@2.4.2)
└── js-yaml@2.0.5 (argparse@0.1.16, esprima@1.0.4)

bootstrap@3.3.5 node_modules/bootstrap

jade@1.11.0 node_modules/jade
├── character-parser@1.2.1
├── void-elements@2.0.1
├── commander@2.6.0
├── mkdirp@0.5.1 (minimist@0.0.8)
├── jstransformer@0.0.2 (is-promise@2.0.0, promise@6.1.0)
├── with@4.0.3 (acorn-globals@1.0.4, acorn@1.2.2)
├── constantinople@3.0.1 (acorn-globals@1.0.4)
├── uglify-js@2.4.23 (uglify-to-browserify@1.0.2, async@0.2.10, yargs@3.5.4, source-map@0.1.34)
├── transformers@2.1.0 (promise@2.0.0, css@1.0.8, uglify-js@2.2.5)
└── clean-css@3.3.3 (commander@2.8.1, source-map@0.4.2)

helmet@0.7.1 node_modules/helmet
├── ienoopen@0.1.0
├── dont-sniff-mimetype@0.1.0
├── hide-powered-by@0.1.0
├── helmet-crossdomain@0.1.0
├── nocache@0.2.0
├── x-xss-protection@0.1.2
├── frameguard@0.2.1 (lodash.isstring@3.0.0)
├── hpkp@0.1.0 (arraywrap@0.1.0)
├── connect@3.3.5 (utils-merge@1.0.0, parseurl@1.3.0, debug@2.1.3, finalhandler@0.3.4)
├── hsts@0.1.1 (lodash@3.5.0)
└── helmet-csp@0.2.2 (camelize@1.0.0, platform@1.3.0, lodash@3.5.0)

grunt-contrib-uglify@0.9.1 node_modules/grunt-contrib-uglify
├── uri-path@0.0.2
├── chalk@1.0.0 (escape-string-regexp@1.0.3, ansi-styles@2.0.1, supports-color@1.3.1, strip-ansi@2.0.1, has-ansi@1.0.3)
├── uglify-js@2.4.23 (uglify-to-browserify@1.0.2, async@0.2.10, yargs@3.5.4, source-map@0.1.34)
├── maxmin@1.1.0 (figures@1.3.5, pretty-bytes@1.0.4, gzip-size@1.0.0)
└── lodash@3.9.3

grunt-nodemon@0.4.0 node_modules/grunt-nodemon
└── nodemon@1.3.7 (minimatch@0.3.0, touch@0.0.3, ps-tree@0.0.3, update-notifier@0.3.2)

grunt-contrib-less@1.0.1 node_modules/grunt-contrib-less
├── chalk@0.5.1 (ansi-styles@1.1.0, escape-string-regexp@1.0.3, supports-color@0.2.0, has-ansi@0.1.0, strip-ansi@0.3.0)
├── lodash@2.4.2
└── less@2.4.0 (graceful-fs@3.0.8, mime@1.3.4, image-size@0.3.5, promise@6.1.0, errno@0.1.2, mkdirp@0.5.1, source-map@0.2.0, request@2.58.0)

grunt-contrib-jshint@0.11.2 node_modules/grunt-contrib-jshint
├── hooker@0.2.3
└── jshint@2.8.0 (strip-json-comments@1.0.2, exit@0.1.2, console-browserify@1.1.0, shelljs@0.3.0, minimatch@2.0.8, cli@0.6.6, htmlparser2@3.8.3, lodash@3.7.0)

connect-mongo@0.8.1 node_modules/connect-mongo
├── depd@1.0.1
├── debug@2.2.0 (ms@0.7.1)
├── lodash@3.9.3
└── mongodb@2.0.33 (readable-stream@1.0.31, mongodb-core@1.1.32)

mongoose@4.0.5 node_modules/mongoose
├── regexp-clone@0.0.1
├── sliced@0.0.5
├── muri@1.0.0
├── hooks-fixed@1.0.1
├── mpromise@0.5.4
├── kareem@1.0.1
├── async@0.9.0
├── mpath@0.1.1
├── ms@0.1.0
├── mquery@1.5.1 (debug@0.7.4, bluebird@2.9.9)
├── mongodb@2.0.33 (readable-stream@1.0.31, mongodb-core@1.1.32)
└── bson@0.3.2 (bson-ext@0.1.8)
```

>npm start

```
> Drywall@0.16.1 start /home/squallssck/drywall
> grunt

Running "copy:vendor" (copy) task
Created 5 directories, copied 109 files

Running "newer:uglify" (newer) task

Running "newer:uglify:layouts" (newer) task

Running "uglify:layouts" (uglify) task
>> 3 sourcemaps created.
>> 3 files created.

Running "newer-postrun:uglify:layouts:1:/home/squallssck/drywall/node_modules/grunt-newer/.cache" (newer-postrun) task

Running "newer:uglify:views" (newer) task

Running "uglify:views" (uglify) task
>> 21 sourcemaps created.
>> 21 files created.

Running "newer-postrun:uglify:views:2:/home/squallssck/drywall/node_modules/grunt-newer/.cache" (newer-postrun) task

Running "newer:less" (newer) task

Running "newer:less:layouts" (newer) task

Running "less:layouts" (less) task
>> NameError: variable @btn-border-radius-base is undefined in public/vendor/bootstrap/less/buttons.less on line 20, column 102:
>> 19   white-space: nowrap;
>> 20   .button-size(@padding-base-vertical; @padding-base-horizontal; @font-size-base; @line-height-base; @btn-border-radius-base);
>> 21   .user-select(none);
Warning: Error compiling public/less/bootstrap-build.less Use --force to continue.

Aborted due to warnings.

npm ERR! Linux 2.6.32-042stab108.2
npm ERR! argv "node" "/usr/bin/npm" "start"
npm ERR! node v0.12.4
npm ERR! npm  v2.11.2
npm ERR! code ELIFECYCLE
npm ERR! Drywall@0.16.1 start: `grunt`
npm ERR! Exit status 6
npm ERR! 
npm ERR! Failed at the Drywall@0.16.1 start script 'grunt'.
npm ERR! This is most likely a problem with the Drywall package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt
npm ERR! You can get their info via:
npm ERR!     npm owner ls Drywall
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/squallssck/drywall/npm-debug.log
```

I saw some similar issue here but there's no conclusion  and I tried different things but still, it's not working. where I am doing wrong? 

btw, I am operating on a Ubuntu 14.04 kiwi vm vps.

--- a/public/less/bootstrap-vars.less
+++ b/public/less/bootstrap-vars.less
@@ -170,6 +170,11 @@
 
 @btn-link-disabled-color:        @gray-light;
 
+// Allows for customizing button radius independently from global border radius
+@btn-border-radius-base:         @border-radius-base;
+@btn-border-radius-large:        @border-radius-large;
+@btn-border-radius-small:        @border-radius-small;
+
 
 //== Forms
 //
@@ -383,7 +388,7 @@
 @navbar-default-toggle-border-color:       #ddd;
 
 
-// Inverted navbar
+//=== Inverted navbar
 // Reset inverted navbar basics
 @navbar-inverse-color:                      lighten(@gray-light, 15%);
 @navbar-inverse-bg:                         #222;
@@ -484,6 +489,7 @@
 @jumbotron-bg:                   @gray-lighter;
 @jumbotron-heading-color:        inherit;
 @jumbotron-font-size:            ceil((@font-size-base * 1.5));
+@jumbotron-heading-font-size:    ceil((@font-size-base * 4.5));
 
 
 //== Form states and alerts

